dsalgo/math.py

```diff
@@ -0,0 +1,197 @@
+from math import gcd, sqrt
+from functools import reduce
+
+
+#Call the following functions according to number of arguments :
+
+#list_hcf(l) - HCF of a list
+#list_prime(l) - Primes in a list
+#next_prime(n) - next prime of a number
+#prev_prime(n) - previous prime of a number
+#list_lcm(a) - LCM of a list
+#hcf_k_elements(l,k) - HCF of k elements in a list
+#lcm_k_elements(l, k) - LCM of k elements in a list
```

Comment:
@keshavgbpecdelhi add these single-line comments in function themselves or create a docstring for them